ci/windows/build_common.psm1

```diff
@@ -0,0 +1,67 @@
+
+param ($CXX_STANDARD,
+       $GPU_ARCHS)
+
+# We need the full path to cl because otherwise cmake will replace CMAKE_CXX_COMPILER with the full path
+# and keep CMAKE_CUDA_HOST_COMPILER at "cl" which breaks our cmake script
+$script:HOST_COMPILER  = (Get-Command "cl").source -replace '\\','/'
+$script:PARALLEL_LEVEL = (Get-WmiObject -class Win32_processor).NumberOfLogicalProcessors
+
+If($null -eq $DEVCONTAINER_NAME) {
```

Comment:
What's the intention? DEVCONTAINER_NAME is set by the user prior to using the scripts? If so, maybe it should be an optional parameter since we'll be launching the scripts manually like this and won't have much opportunity for setting powershell variables:

`docker run (blah.blah...) /mount/ci/windows/build_libcudacxx.ps1 arg1 arg2`

If we use `$ENV:DEVCONTAINER_NAME` I'd be more amicable to the idea as we could set it on launch.

Reply:
```suggestion
If($null -eq $ENV:DEVCONTAINER_NAME) {
```